### DISCUSSION TITLE
Can you make it remember the position/track you were on when you switched project tabs?

### DISCUSSION THREAD

**Discussion by Sy256** [helge17/tuxguitar#81] [General]
Real low priority, but right now it seems to revert back to the beginning and top track of whatever project tab you switch to, making it not so easy to compare certain things between projects at the same spots. Could you set it to remember that view somehow? Thanks!

This is another thing 2.0 beta did, so maybe you can pull it straight from there or it can give clues how it worked?

**Comment by guiv42** [comment]
done (#108)

**Reply by Sy256** [reply]
Seems to be working here! Thank you, closed!
